src/cmds/service.c

```diff
@@ -11,15 +11,69 @@
 #include <string.h>
 #include <unistd.h>
 #include <stdlib.h>
+#include <mem/misc/pool.h>
+#include <util/dlist.h>
+#include <sys/types.h>
+#include <signal.h>
 
-static void srv_print_services(void) {
-	printf("PRINT LIST OF SERVICES IS NOT SUPPORTED YET\n");
+#define len 32
+
+struct service_list {
+	struct dlist_head dlist_item;
+	char service_name[len];
+	pid_t pid;
+};
+
+POOL_DEF(pool, struct service_list, sizeof(struct service_list));		 
```

Comment:
Third argument of POOL_DEF is a number of pool objects, so `sizeof struct` has no sense here.